src/Avalonia.Base/AvaloniaObject.cs

```diff
@@ -223,9 +210,9 @@ public object GetValue(AvaloniaProperty property)
             {
                 return ((IDirectPropertyAccessor)GetRegistered(property)).GetValue(this);
             }
-            else if (_values != null)
+            else if (Values != null)
```

Comment:
`Values` can never be null can it? I think this should be changed back to `_values`.